homeassistant/components/advantage_air/climate.py

```diff
@@ -220,6 +224,9 @@ async def async_set_temperature(self, **kwargs):
 
     async def set_myzone(self, **kwargs):
         """Set this zone as the 'MyZone'."""
+        _LOGGER.warning(
+            "This service has been depreciated and will be removed in a future version, please use select.select_option"
```

Comment:
```suggestion
            "This service has been deprecated and will be removed in a future version, please use select.select_option"
```

Reply:
Should we tell them the name of the service in case they look at the logs later?